frontend/src/features/projects/spl/steps/SplStep.tsx

```diff
@@ -83,8 +83,8 @@ const SplStep = ({ formikRef }: IStepProps) => {
       statusCode: DisposeWorkflowStatus.Draft,
     },
     {
-      schema: SurplusPropertyListNotInSplYupSchema,
```

Comment:
it looks like the not in spl schema on the erp tab got replaced with this new schema on the spl tab, is that what is desired?

Reply:
When removing from SPL there are only 3 required fields, `Request for removal on`, `Request for removal approved on` and `Rationale for removal`.  The original validation was looking for a field on the ERP tab.